election/views.py

```diff
@@ -47,15 +49,23 @@ def send_mail_invitation(
         activate(election.select_language)
 
     text_body = render_to_string("election/mail_invitation.txt", merge_data)
-    html_body = render_to_string("election/mail_invitation.html", merge_data)   
-
-    msg = EmailMultiAlternatives(
-        f"[{gettext('Mieux Voter')}] {election.title}",
-        text_body,
-        settings.EMAIL_HOST_USER,
-        [email])
-    msg.attach_alternative(html_body, "text/html")
-    msg.send()
+    html_body = render_to_string("election/mail_invitation.html", merge_data)
+
+    send_status = requests.post(
```

Comment:
This line introduces a new dependency with `requests`. Whereas `requests` fills a gap, I feel it's a bit overkill not using `urllib` here. Also, since we don't address Python2, `urllib` is fairly simple to use,